envoy-control-tests/src/main/kotlin/pl/allegro/tech/servicemesh/envoycontrol/config/BaseEnvoyTest.kt

```diff
@@ -48,6 +50,7 @@ open class BaseEnvoyTest {
         init {
             echoContainer.start()
             echoContainer2.start()
+            httpsEchoContainer.start()
```

Comment:
I think we shouldn't add yet another container that starts in every test, even if it is not used in 95% of the tests. It should be started (and stopped) in a test which actually use it.

BTW. We added a command for starting the container, but we didn't add a command for stoping it.